study_group_app/lib/screens/auth/login.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:flutter/material.dart';
+//import 'package:google_sign_in/google_sign_in.dart';
+//import 'package:firebase_auth/firebase_auth.dart';
+
+// Initializes GoogleSignIn & FirebaseAuth
+//final GoogleSignIn _googleSignIn = GoogleSignIn();
+//final FirebaseAuth _auth = FirebaseAuth.instance;
+
+// Stateful Login Page Class
+class LoginPage extends StatefulWidget {
+  LoginPage({Key key, this.title}) : super(key: key);
+  final String title;
+
+  // Creates the stateful widget LoginPage
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  // Main build function, generates the view
+  @override
+  Widget build(BuildContext context) {
+    // Scaffold is from MaterialApp. Implements the basic visual layout
+    return Scaffold(
+      appBar: AppBar(
+        // Top bar, title is set using widget.title which is passed down from main
+        title: Text(widget.title),
+      ),
+      // Container: UI for Login Form
+      body: Container(
+        padding: EdgeInsets.fromLTRB(20, 300, 20, 40),
```

Comment:
I'm getting bottom overflow from this when the keyboard is brought up. Could change it to this to fix that issue and then we can reposition the text boxes later if we want. 
```suggestion
        padding: EdgeInsets.all(20),
```